src/app/components/secret-key/mnemonic-key/mnemonic-word-input.tsx

```diff
@@ -1,8 +1,10 @@
-import { wordlist } from '@scure/bip39/wordlists/english';
+import { useState } from 'react';
```

Comment:
I was trying to figure out how this worked without `wordlist` but realised it wasn't actually being used here. Nice refactor 👍